staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -751,11 +747,11 @@ func (s *store) GetList(ctx context.Context, key string, opts storage.ListOption
 			getResp.Kvs[i] = nil
 		}
 
-		// no more results remain or we didn't request paging
-		if !hasMore || !paging {
```

Comment:
same comment about the bool being relevant

Reply:
done